src/api/orderProducts/orderProducts.service.js

```diff
@@ -6,7 +6,16 @@ async function getProductGameTitleFromOrder({ orderId }) {
   return productGameTitleFromOrder;
 }
 
+async function log({ orderId, products }) {
+  for (let i = 0; i < products.length; i++) {
+    const product = products[i];
+    product.orderId = orderId;
+  }
```

Comment:
What's the purpose of lines 9 to 13? I see a declared 'product' const, but you don't use it later.